content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -300,7 +294,7 @@ spec:
     storage: 5Gi
   volumeMode: Filesystem
   accessModes:
-    - ReadWriteOnce
+    - `Once
```

Comment:
Is this correct?

Reply:
and there is no access mode called `ReadWriteOnce`.